src/docs/references/marketplace-assets/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Marketplace assets
+slug: marketplace-assets
+updated: 2022-05-09
+category: references
+ingress:
+  Reference documentation providing information on Marketplace assets.
+published: true
+---
+
+_Marketplace assets_ are a mechanism defining configuration data and
+content for a marketplace. They are most useful for storing client
+application configuration and content that is managed by a marketplace
+operator and changes relatively infrequently. Each _marketplace asset_
+is an object that defines a _path_ and _content_. The path gives the
+asset a name and allows organizing assets in a way similar to a file
+system, where assets can be organized in directory-like structure. The
+content of the asset is its data in one of the supported data types.
+
+At present, Flex supports only JSON data as assets. Support for images
+(JPEG, PNG, etc) is coming in the future.
+
+The marketplace assets are typically edited by a marketplace operator
+though Flex Console.
+
+**IMPORTANT** All marketplace assets are considered public and must not
+be used to store secret or otherwise sensitive information.
+
+> **Note:** Marketplace assets are in early development. At present,
+> Console allows only limited ability to manage the content of a single
+> marketplace asset - the JSON document describing the
+> [translation strings](/concepts/translations/) for a client
+> application. Stay tuned for more later on!
+
+For example, a marketplace may have assets with the following paths:
+
+```
+config/config.json
+config/translations.json
+```
+
+where `config/config.json` could contain configuration data (for
+instance, UI colors, marketplace pricing settings, etc) and
+`config/translations.json` could contain string translation data for use
+in a marketplace client application (such as one based on one of the FTW
+templates).
+
+## Asset versioning
```

Comment:
Would it make sense to say here that asset tree version is analogous to Git SHA? People are familiar with Git so maybe that would make it clear what the tree version is